doc/api/http.md

```diff
@@ -608,6 +608,12 @@ const setCookie = request.getHeader('set-cookie');
 
 ```
 
+### request.maxHeadersCount
+
+* {number} **Default:** `2000`
+
+Limits maximum response headers count. If set to 0 - no limit will be applied.
```

Comment:
Shouldn't it be a comma instead of a hyphen ? Same for [server.maxHeadersCount](https://github.com/darai0512/node/blob/9ea8baeff28b10d3c55a621b7974c17ca252d1ae/doc/api/http.md#servermaxheaderscount)

> If set to 0, no limit will be applied.

Reply:
Thanks for your review, PTAL.

Reply:
@Trott : Can you please confirm if above is fine. Missed to cc you in the initial comment. Thanks !